docs/docsite/rst/porting_guides/porting_guide_core_2.19.rst

```diff
@@ -29,6 +29,7 @@ with wide-ranging positive effects on security, performance, and user experience
 Backward compatibility has been preserved where practical, but some breaking changes were necessary.
 This guide describes some common problem scenarios with example content, error messsages, and suggested solutions.
 
+We recommend you test your playbooks and roles in a staging environment with stable-2.19 to determine where you may need to make changes.
```

Comment:
It might be a good idea to show an example of how people can "install" stable-2.19?